server/pbench/lib/mappings/grafana/templates/block_device.json

```diff
@@ -0,0 +1,259 @@
+{
```

Comment:
It seems like we could merge all these templates into a form shared across the various layouts, removing lots of duplication.  Can we look into that?